sample-xml/app/src/main/java/co/nimblehq/sample/xml/ui/screens/home/HomeFragment.kt

```diff
@@ -18,19 +19,29 @@ class HomeFragment : BaseFragment<FragmentHomeBinding>() {
     lateinit var navigator: MainNavigator
 
     private val viewModel: HomeViewModel by provideViewModels()
+    private val itemListAdapter: ItemListAdapter by lazy { ItemListAdapter() }
 
     override val bindingInflater: (LayoutInflater, ViewGroup?, Boolean) -> FragmentHomeBinding
         get() = { inflater, container, attachToParent ->
             FragmentHomeBinding.inflate(inflater, container, attachToParent)
         }
 
+    override fun setupView() {
+        binding.rvHome.apply {
+            adapter = itemListAdapter.apply {
+                onItemClicked = { uiModel -> viewModel.navigateToSecond(uiModel) }
+            }
+            addItemDecoration(ItemDivider(this.context))
```

Comment:
@ryan-conway At first, I used this way. But as @chornerman update the logic to add the decoration if it's not the last item on the `sample-compose`. So I have no choice. 😂

Reply:
Or could we omit the last-item-checking logic on `sample-compose`? @ryan-conway @chornerman  So that we can have a simple divider here.

Reply:
@luongvo, I think let's make it consistent with `sample-compose` because one of the purposes that we separate the `folder` is to show how to build the `UI` differently between `XML and Compose,` So the people can see how it was different in both ways. 🤔 

Also, IMO I think this's a good point to have it as we're seen the way to handle the `Divider` on the last item is so different from both ways; in `compose` is so easy to handle than `XML,` but if we don't handle the last item of `Divider` for `XML` folder, then we should update in `Compose` too. 🙇‍♀️ cc @lydiasama @ryan-conway @chornerman

Reply:
Actually, I agree with @luongvo to keep the logic to be simple, but the point to compare between `XML and Compose` as @Wadeewee raised is good too. Finally, I ended up to be siding to keep the logic to be simple instead. 🙏🏻 
This could require your help @chornerman . Could you please open one more PR for the same issue #242  to update this to be consistent? 🙏🏻

Reply:
Agreed with @luongvo and @lydiasama 👍🏻

Reply:
@lydiasama Will do 👍🏼